app/src/main/java/org/mozilla/tv/firefox/architecture/FirefoxViewModelProviders.kt

```diff
@@ -17,11 +17,11 @@ import org.mozilla.tv.firefox.ext.serviceLocator
  */
 object FirefoxViewModelProviders {
 
-    fun of(activity: FragmentActivity): ViewModelProvider {
+    fun of(activity: androidx.fragment.app.FragmentActivity): ViewModelProvider {
```

Comment:
Looks like a Android Studio build residue. I think this happens if there are more declarations of same class names from different packages - which shouldn't be the case anymore? Try clean, rebuild and re-import?